### ISSUE TITLE
List: aria-posinset and aria-setsize are wrong in this case

### ISSUE THREAD

**Issue by bpasero** [Microsoft/vscode#44177]
Steps to Reproduce:

1. have a HTML file to trigger notifications like the one below
2. follow one link to get an error that the file does not exist and now open notifications center to get a list view
3. continue with other links until you get 3-4 errors

=> check the `aria-posinset` and `aria-setsize` property having weird values

![image](https://user-images.githubusercontent.com/900690/36523920-38e9d346-17a4-11e8-9793-0dadf2392be3.png)

```html
<!DOCTYPE html>
<html>

<head>
	<title>Mankala</title>
    <meta http-equiv="X-UA-Compatible" content="IE=Edge"/>
    <script type="text/javascript" src="Game2.js"></script>
  	<script type="text/javascript" src="Driver2.js"></script>
	<script type="text/javascript" src="Features2.js"></script>
	<script type="text/javascript" src="Position.js"></script>
	<script type="text/javascript" src="lib/Geometry.js"></script>
	<script type="text/javascript" src="lib/Base.js"></script>
</head>

<body id="bod" onload="Mankala.testBrowser()">
   <svg height="10px">
     <defs>
       <radialGradient id="grad1" cx="50%" cy="50%" r="50%" fx="50%" fy="50%">
         <stop offset="0%" style="stop-color:aqua;
                                  stop-opacity:0.35" />
         <stop offset="100%" style="stop-color:rgb(0,0,255);stop-opacity:1" />
       </radialGradient>
       <radialGradient id="grad2" cx="50%" cy="50%" r="50%" fx="50%" fy="50%">
         <stop offset="0%" style="stop-color:gold;
                                  stop-opacity:0.35" />
         <stop offset="100%" style="stop-color:rgb(0,255,0);stop-opacity:1" />
       </radialGradient>
     </defs>
   </svg>
</body>
</html>
```

**Comment by joaomoreno** [comment]
Yeah, [this](https://github.com/Microsoft/vscode/blob/master/src/vs/base/browser/ui/list/listWidget.ts#L206) doesn't work. We need to move aria down to the ListView.

**Comment by bpasero** [comment]
@joaomoreno thanks!
